kickstarter/android-oss
# Horizontal Rewards Design Sweep

# What ❓
- Completed by @Scollaco and myself
- We added the `RewardItems` back into the rewards so you can now see them. 
- Also we fixed added the `RewardItemDecorator` class for the divider line between reward items. Previously we were inserting views in the `rewards_item_view.xml`.
- Updated strings from master.

# How to QA? 🤔
- When your feature flag is enabled and you go to view `Rewards` you will see them as expected compared to the final designs. 

# Story 📖
[Horizontal Rewards Design Sweep](https://trello.com/c/D54LycLf/1324-horizontal-rewards-design-sweep)

# See 👀
<img src=https://user-images.githubusercontent.com/16387538/56138195-63f6d580-5f64-11e9-9e5e-6627f49b59c6.jpg width=200/><img src=https://user-images.githubusercontent.com/16387538/56138198-65280280-5f64-11e9-8038-61b6f5f7360d.jpg width=200/>  <img src=https://user-images.githubusercontent.com/16387538/56145421-46306d00-5f72-11e9-97c3-79681217fa8a.jpg width=200/>
